woocommerce/woocommerce-gutenberg-products-block
# Decode address entities in shipping calculator

### Screenshots

_Before:_
![imatge](https://user-images.githubusercontent.com/3616980/77325726-14468f80-6d19-11ea-9979-dc15f1160563.png)

_After:_
![imatge](https://user-images.githubusercontent.com/3616980/77325771-21fc1500-6d19-11ea-9dbc-41390ea31b59.png)

### How to test the changes in this Pull Request:

1. Go to the _Cart_ block and add a city with a special character in the shipping calculator.
2. Update the address and reload the page.
3. Verify the address appears correctly.
